src/components/PathwayCourse/PathwayCards/index.js

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect } from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { breakpoints } from "../../../theme/constant";
+import Chip from "@mui/material/Chip";
+import { lang as language } from "../../../constant";
+import {
+  Container,
+  Box,
+  Grid,
+  Card,
+  Button,
+  CardContent,
+  Typography,
+} from "@mui/material";
+import { interpolatePath, PATHS } from "../../../constant";
+import { format } from "../../../common/date";
+import { getCourseContent } from "../../Course/redux/api";
+import { useSelector } from "react-redux";
+import { versionCode } from "../../../constant";
+import { useHistory } from "react-router-dom";
+const PathwayCards = (props) => {
+  // const language = {
+  //   hi: "Hindi",
+  //   en: "English",
+  //   mr: "Marathi",
+  // };
+
+  // const language ;
+
+  const history = useHistory();
+  const language = {
+    hi: "Hindi",
+    en: "English",
+    mr: "Marathi",
+  };
+  const { userEnrolledClasses, data } = props;
+  console.log(userEnrolledClasses);
+  function UpcomingClassCardComponent({ item }) {
+    const user = useSelector(({ User }) => User);
+    const [classIndex, setClassIndex] = React.useState(0);
+    useEffect(() => {
+      const courseId = item.course_id;
+      getCourseContent({ courseId, versionCode, user }).then((res) => {
+        console.log(
+          res.data.course.exercises.forEach((ex, index) => {
+            if (ex.id === item.exercise_id) {
+              setClassIndex(index + 1);
+              return;
+            }
+          })
+        );
+      });
+    }, []);
+    return (
+      <Grid item xs={12} sm={4} md={4}>
+        <Card
+          onClick={() => {
+            console.log("clicked");
+            history.push(
+              interpolatePath(PATHS.PATHWAY_COURSE_CONTENT, {
+                courseId: item.course_id,
+                exerciseId: classIndex,
+                pathwayId: item.pathway_id,
+              })
+            );
+          }}
+          style={{ minWidth: "300px", margin: "10px" }}
+        >
+          {item.type == "batch" ? (
+            <Box sx={{ borderTop: 5, color: "ForestGreen" }} />
+          ) : (
+            <Box sx={{ borderTop: 5, color: "darkblue" }} />
+          )}
```

Comment:
Use `===` when comparing items expected to be the same type (`string` in this case), stay consistent with lowercase/uppercase (in this case use colors in all lowercase letters), and use the conditional operator only where there are differences.  An exception to the last guideline would be if you expect additional differences in the future (e.g., if in the future `borderTop` might be 5 or 10 depending on whether it's `batch` or not.)  
```suggestion
          <Box sx={{ borderTop: 5, color: item.type === "batch" ? "forestgreen" : "darkblue" }} />
```